benchmarks/BM_efficientnet/benchmark.sh

```diff
@@ -0,0 +1,66 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
+# this software and associated documentation files (the "Software"), to deal in
+# the Software without restriction, including without limitation the rights to
+# use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+# the Software, and to permit persons to whom the Software is furnished to do so,
+# subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+# IMPORTANT: $3 argument is used for the CI
+
+source setup.sh
+
+pushd ../.. || exit 1  # Repo's root directory
+
+pushd docs/examples/efficientnet || exit 1
+
+is_number_re='^[0-9]+$'
+
+if [[ "$1" == "do_setup" ]]; then
+  if ! [[ $2 =~ $is_number_re ]]; then
+    echo "Please provide the max batch size."
+  else
+    echo "Configuring inference with max batch size: $2"
+  fi
+  source setup.sh "$2"
+fi
+
+if [[ -z "$3" ]]; then
+    DOCKER_RUN_ARGS="--gpus all --net host -p8000:8000 -p8001:8001 -p8002:8002 --privileged"
```

Comment:
Do you want to make it use `--gpus all` bye default?
If so I would extend the `sleep 10` to `sleep 20` as it takes more time to initialize 8 GPUs than 1.

Reply:
Good point.
Yes, I believe that running all GPUs by default is a good idea - by default Triton will create one model instance per GPU in this situation.
Changed `sleep` to 20.